marshal/bigint/unmarshal.go

```diff
@@ -71,7 +71,7 @@ func Unmarshal(data []byte, value interface{}) error {
 		rv := reflect.ValueOf(value)
 		rt := rv.Type()
 		if rt.Kind() != reflect.Ptr {
-			return fmt.Errorf("failed to unmarshal bigint: unsupported value type (%T)(%#[1]v)", value)
+			return fmt.Errorf("failed to unmarshal bigint: unsupported value type %#v", value)
```

Comment:
Why did you change it ? now you don't see type in the error message

Reply:
`%#v` - include type